docs/server/source/production-deployment-template/architecture.rst

```diff
@@ -0,0 +1,69 @@
+Architecture of an IPDB Node
+============================
+
+An IPDB Production deployment is hosted on a Kubernetes cluster and includes:
+
+* NGINX, OpenResty, BigchainDB and MongoDB
+  `Kubernetes Services <https://kubernetes.io/docs/concepts/services-networking/service/>`_.
+* NGINX, OpenResty, BigchainDB, Monitoring Agent and Backup Agent
+  `Kubernetes Deployments <https://kubernetes.io/docs/concepts/workloads/controllers/deployment/>`_.
```

Comment:
is mongoldb deployment just a proxy service since IPDB is using hosted mongoldb cloud?

Reply:
We do not use hosted MongoDB cloud for running MongoDB.
We only leverage MongoDB Cloud Manager for backups.